docs/modules/reference/pages/configuration/ttl-rpc.adoc

```diff
@@ -8,7 +9,7 @@ Our modules and services have different rules for determining the TTL.
 
 TIP: To troubleshoot TTLs, consider enabling the Open Tracing integration in {page-component-title} Core instance and the Minion.
 
-== TTLs in Pollerd & Collectd
+== TTLs in pollerd and collectd
```

Comment:
Not a blocker, but aren't "Pollerd" and "Collectd" proper nouns? They're the names of the daemons, and how they're referred to in, say, configuration files and such. I would have assumed all the Somethingd's would stay captialized.